ruleset/sca/darwin/18/cis_apple_macOS_10.14.yml

```diff
@@ -0,0 +1,885 @@
+    rationale: "Disabling automatic login decreases the likelihood of an unauthorized person gaining access to a system."
+    remediation: "Run the following command in Terminal: sudo defaults delete /Library/Preferences/com.apple.loginwindow autoLoginUser"
+    compliance:
+      - cis: ["5.8"]
+    condition: none
+    rules:
+      - 'c:defaults read /Library/Preferences/com.apple.loginwindow -> r:autoLoginUser'
+
+# jal 
+# jal not implemented - Not a command - 5.9 Require a password to wake the computer from sleep or screen saver (Manual)
+
+# jal
+# jal not implemented - SCA Limit -> 5.10 Ensure system is set to hibernate (Automated) 
+# jal not implemented - SCA Limit -> 5.11 Require an administrator password to access system-wide preferences (Automated) 
+# jal not implemented - SCA Limit -> 5.12 Disable ability to login to another user's active and locked session (Automated)  
+
+# 5.13 Create a custom message for the Login Screen (Automated) 17058
+  - id: 17058
+    title: "Create a custom message for the Login Screen" 
+    description: "An access warning informs the user that the system is reserved for authorized use only, and that the use of the system may be monitored."
+    rationale: "An access warning may reduce a casual attacker's tendency to target the system. Access warnings may also aid in the prosecution of an attacker by evincing the attacker's knowledge of the system's private status, acceptable use policy, and authorization requirements."
+    remediation: "Run the following command to enable a custom login screen message: sudo defaults write /Library/Preferences/com.apple.loginwindow LoginwindowText \"<custom.message>\""
+    compliance:
+      - cis: ["5.13"]
+    condition: all
+    rules:
+      - 'c:defaults read /Library/Preferences/com.apple.loginwindow.plist LoginwindowText -> !r:does not exist'
+
+# 5.14 Create a Login window banner (Automated) 17059
+  - id: 17059
+    title: "Create a Login window banner" 
+    description: "A Login window banner warning informs the user that the system is reserved for authorized use only. It enforces an acknowledgment by the user that they have been informed of the use policy in the banner if required. The system recognizes either the .txt and the .rtf formats."
+    rationale: "An access warning may reduce a casual attacker's tendency to target the system. Access warnings may also aid in the prosecution of an attacker by evincing the attacker's knowledge of the system's private status, acceptable use policy, and authorization requirements."
+    remediation: "Edit (or create) a PolicyBanner.txt or PolicyBanner.rtf file, in the /Library/Security/ folder, to include the required login window banner text."
+    compliance:
+      - cis: ["5.14"]
+    condition: all
+    rules:
+      - 'c:cat /Library/Security/PolicyBanner.* -> !r:^$'
```

Comment:
This would be replaced by https://documentation.wazuh.com/4.0/user-manual/capabilities/sec-config-assessment/creating_custom_policies.html#rule-syntax-for-directories

```
Check if a directory contains files that match a regex: d:/path/to/directory -> r:^files
```

Could we use the following?
```
- 'd:/Library/Security -> r:^PolicyBanner'
```
It looks for files starting by `PolicyBanner` at `/Library/Security`